template-compose/build.gradle.kts

```diff
@@ -6,15 +6,15 @@ buildscript {
     }
 
     dependencies {
-        classpath("com.android.tools.build:gradle:${Versions.BUILD_GRADLE_VERSION}")
-        classpath("com.google.dagger:hilt-android-gradle-plugin:${Versions.HILT_VERSION}")
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.KOTLIN_VERSION}")
+        classpath("com.android.tools.build:gradle:${Versions.GRADLE}")
+        classpath("com.google.dagger:hilt-android-gradle-plugin:${Versions.HILT}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.KOTLIN}")
```

Comment:
Can we also configure for these dependencies as well?

Reply:
@kaungkhantsoe Nope, this deprecated config [will be removed when migrating to Gradle 8](https://github.com/nimblehq/android-templates/pull/508/files#diff-a86af81bc38d910132ed4be8d8a2ece3b82bb775bdae609344303399f3726afc).